modules/ROOT/pages/dw-core-functions-orderby.adoc

```diff
@@ -40,38 +40,28 @@ array. Note that `orderBy($.letter)` produces the same result as
 %dw 2.0
 output application/json
 ---
-{ letter: "b", letter: "c", letter: "a" } orderBy ((value, key) -> value)
```

Comment:
pls see if wee should restore this or if it was removed from source

I wonder if some manual changes were made to the docs on the site but not in the source or vice versa. Need to figure it out. We intentionally add things to the docs sometimes based on feedback from customers.